docker-compose.yml

```
@@ -33,3 +33,12 @@ services:
       interval: 10s
       timeout: 5s
       retries: 5
+
+  dozzle:
+    image: amir20/dozzle:latest
+    container_name: dozzle
+    ports:
+      - "9999:8080"
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    restart: always
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)